tests/UnitTests/test_view_model.py

```diff
@@ -0,0 +1,100 @@
+"""Unit tests for ViewModel.py"""
+
+from datetime import datetime
+from todo_app.ListUtils import to_utc_datetime
+from todo_app.TrelloItem import TrelloItem
+from todo_app.ViewModel import ViewModel
+import pytest
+
+
+@pytest.fixture
+def view_model() -> ViewModel:
+    items = [TrelloItem('Id01', 'Item Name 01', 'To Do', to_utc_datetime('2021-03-16T12:30:01.070Z')), TrelloItem('Id02', 'Item Name 02', 'Doing', to_utc_datetime('2021-03-15T12:30:01.070Z')), TrelloItem('Id03', 'Item Name 03', 'Done', to_utc_datetime('2021-03-14T12:30:01.070Z'))]
```

Comment:
Note that here you pass through a datetime to your items - we're not doing that in the application though!